source/api_documentation/call_api/_get_call_transcript_caller_agent_conversation.rst

```diff
@@ -0,0 +1,28 @@
+.. container:: endpoint-long-description
+
+  .. rubric:: Examples
+
+  Get transcript for the given call record `CUID-33` with both agent and caller included in the corresponding conversation order.
+
+  Endpoint:
+
+  ``https://mynetwork.invoca.net/call/transcript/CUID-33?transcript_format=agent_caller_conversation``
+
+  Format: application/json
+
+  Response Code: 200
+
+  Response Body:
+
+  .. code-block:: json
+
+     [
+       {
```

Comment:
Same for this example, we pass back each line of text as it's own hash object. See example above for reference.

Reply:
Depending on answer to above.. I wonder if we instead want an array of single hashes?

```
[
  { 
    agent: "Hi",
    caller: "Yo"
  },
  {
    agent: "How can I help you?",
    caller: "Actually, wrong number"
  },
  {
    agent: "Okay. Bye"
  }
]
```

Reply:
Confirmed with Nash, we do want to keep it as is. We know it's not perfectly aligned with existing DT format but he prefers the lines in individual blocks since it seems more consistent with a conversation.

Reply:
```[
  { 
    agent: "Hi"
  },
  {
    caller: "Yo"
  },
  {
    agent: "How can I help you?"
  },
  {
    caller: "Actually, wrong number"
  },
  {
    agent: "Okay. Bye"
  }
]